routes/document/POST.js

```diff
@@ -0,0 +1,35 @@
+import { RouteContext } from "gadget-server";
+import { OpenAIAssistant, Thread } from "../../helpers/OpenAIHelper";
+/**
+ * Route handler for POST chat
+ *
+ * @param { RouteContext } route context - see: https://docs.gadget.dev/guides/http-routes/route-configuration#route-context
+ *
+ */
+export default async function route({
+  request,
+  reply,
+  api,
+  logger,
+  connections,
+}) {
+  const { name, description } = request.body;
```

Comment:
This code is redundant since we aren't taking in JSON data and then converting it into a document. James was able to use the Gadget client.